presto-main/src/main/java/io/prestosql/server/security/AuthenticationFilter.java

```diff
@@ -121,7 +124,11 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
         if (messages.isEmpty()) {
             messages.add("Unauthorized");
         }
-        response.sendError(SC_UNAUTHORIZED, Joiner.on(" | ").join(messages));
+        response.setStatus(SC_UNAUTHORIZED, Joiner.on(" | ").join(messages));
+        response.setContentType(PLAIN_TEXT_UTF_8.toString());
+        try (PrintWriter writer = response.getWriter()) {
+            writer.write(Joiner.on(" | ").join(messages));
```

Comment:
Here you can write them in separate lines. Or even separated with `\n\n`

Reply:
The client currently uses the HTTP status message as the error message, but since that’s gone in HTTP/2, we need to move to the body. It seems best to keep the same format. The error goes into an exception message, so we want it to be a single line.

We could separate with blank lines here and move it to the client.

Reply:
okay, i'm fine with this as-is. Add a comment, otherwise it looks like for-humans only.